tron/core/actionrun.py

```diff
@@ -776,7 +776,11 @@ def handle_unknown(self):
         desired_delay = INITIAL_RECOVER_DELAY * (3 ** self.recover_tries)
         self.recover_tries += 1
         log.info(f'Starting try #{self.recover_tries} to recover {self.id}, waiting {desired_delay}')
-        return self.do_recover(delay=desired_delay)
+        try:
+            return self.do_recover(delay=desired_delay)
+        except Exception as e:
+            log.error(f'Could not recover {self.id}: {str(e)}')
```

Comment:
Here and everywhere else: `str(e)` strips a lot of information from the exception, including the exception type and the stack trace.
Could you please fix that.